packages/docusaurus-plugin-content-blog/src/index.ts

```diff
@@ -41,6 +41,18 @@ const DEFAULT_OPTIONS: PluginOptions = {
   truncateMarker: /<!--\s*(truncate)\s*-->/, // string or regex
 };
 
+const getFeedTypes = ({type}: {type: string}) => {
```

Comment:
```suggestion
const getFeedTypes = ({type}: {type: 'rss' | 'atom' | 'all'}) => {
```

https://github.com/facebook/docusaurus/blob/61551152805aa53b1a50abf307ed6491ff3ab742/packages/docusaurus-plugin-content-blog/src/types.ts#L33

Let's be stricter, and throw error if some bad user use other string. eg: 'superman'

Reply:
I guess this is a dumb question, but why doesn't work something like this?

```
const getFeedTypes = ({type}: {type: PluginOptions.feedOptions.type) => {
```

Reply:
Fixed, but be that as it may, I would change the code in new PR to avoid such duplication...